stories/card/card.stories.js

```diff
@@ -724,19 +712,23 @@ export const objectCard = () => `
     </div>
 </div>
 `;
+
+objectCard.storyName = 'Object card';
 objectCard.parameters = {
     docs: {
         iframeHeight: 600,
-        storyDescription: `The Object Card Header is the same specified in the Card Anatomy. The Content has a 1rem padding applied on all sides. The <code>.fd-card--object</code> modifier class needs to be applied with <code>.fd-card</code>.
+        storyDescription: `The object card can display a single object or a group of objects. Contrary to other cards, object card has 1rem padding on all sides of the content area. To display an object card, add the <code>fd-card--object</code> modifier class to the main element.
```

Comment:
Ah! I saw this after my comment on Table card. This is perfect :) Could you make Table card last sentence similar to this one?

Reply:
Done.